.github/workflows/publish-dual.yml

```
@@ -0,0 +1,128 @@
+          fs.writeFileSync('package.json', JSON.stringify(pkg, null, 2));
+          "
+          
+          npm publish
+          
+          # Restore original package.json
+          mv package.json.backup package.json
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create summary
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:48: Double quote to prevent globbing and word splitting

(shellcheck)